yarn.lock

```diff
@@ -26007,6 +26057,11 @@ typescript@~3.7.2:
   resolved "https://registry.yarnpkg.com/typescript/-/typescript-3.7.2.tgz#27e489b95fa5909445e9fef5ee48d81697ad18fb"
   integrity sha512-ml7V7JfiN2Xwvcer+XAf2csGO1bPBdRbFCkYBczNZggrBZ9c7G3riSUeJmqEU5uOtXNPMhE3n+R4FA/3YOAWOQ==
 
+typescript@~4.3.2:
```

Comment:
This is probably being pulled in because API Extractor is tied to a specific TS version, which means that we'd be generating API docs and compiling with different versions (which is very likely to cause issues at some point). I'm curious what happens if we force API Extractor to resolve to TS 4.1.5 instead.